zebrad/src/components/mempool.rs

```diff
@@ -88,6 +91,30 @@ enum ActiveState {
     },
 }
 
+/// Mempool configuration section.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Config {
+    /// The transaction cost limit
+    pub tx_cost_limit: u32,
+    /// Max amount of minutes for transactions to be in recently evicted
+    pub eviction_memory_minutes: Duration,
```

Comment:
Variable name: It's just a time in `Duration` format, not an integer number of minutes.

```suggestion
    pub eviction_memory_time: Duration,
```